packages/discovery-react-components/src/components/DocumentPreview/components/PdfViewerWithHighlight/PdfViewerWithHighlight.tsx

```diff
@@ -99,32 +101,51 @@ const PdfViewerWithHighlight = forwardRef<any, Props>(
 
     const setCurrentErrMsgFromPdfConst = useIsPfdError(_isPdfRenderError, setIsPdfRenderError);
 
+    // TODO: Dynamically create a style for every category. Match color of category
+    const styleColor = Object.values(facetInfoMap || {})
+      .map(facetInfo => {
+        return `
+        .category_${facetInfo.facetId}.mf-highlight {
+          background: ${facetInfo.color};
+          border: 2px solid ${facetInfo.color};
```

Comment:
If we're setting the background to `facetInfo.color`, why do we also need a border of the same color?

Reply:
Also, if we wanted to set opacity to 20% as the design spec calls for, we should do that here I think

Reply:
Is the second comment about opacity? That is set to 20% in `.bx--document-preview-pdf-viewer-highlight__item`

![image](https://github.com/watson-developer-cloud/discovery-components/assets/45341518/b4fd5ae2-014a-4152-83c5-6c5234cca9a2)